executor/index_lookup_join.go

```diff
@@ -103,6 +107,8 @@ type lookUpJoinTask struct {
 type outerWorker struct {
 	outerCtx
 
+	lookup *IndexLookUpJoin
```

Comment:
Why not only save the `isOuterJoin` and move it into the `outerCtx`?

Reply:
updated.

Reply:
`outerWorker` also needs `IndexLookUpJoin.requiredRows`, so it maybe more clearer than put these two variables `RequiredRows` and `IsOuterJoin` into `outerCtx`.